### ISSUE TITLE
MIT license violation

### ISSUE THREAD

**Issue by djmattyg007** [luis-almeida/unveil#105]
This project claims to be under the MIT license, but I believe this project is not complying with the terms of the license.

The terms of the license contain the following clause:

```
The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```

I can't find any copy of the copyright notice contained within the terms of the MIT license text within this git repository. I would recommend that this is rectified immediately.

**Comment by hongaar** [comment]
This repository is not a **copy of the software** (one could argue it is, strictly speaking, but I suppose not as meant in the license text), so I don't see the problem here.

**Comment by djmattyg007** [comment]
That fundamentally doesn't make sense, but even if I went by your definition, if I click the `Download ZIP` button in the top-right corner of the GitHub repository viewer, or clone this repository with `git`, I'll obtain a **copy of the software** that doesn't have the license text that is arguably being distributed by the maintainer of the repository.

**Comment by hongaar** [comment]
Indeed, by including the licence in the original source, the maintainer ensures that all copies will have the permission notice included, too. And I'd agree with you that it'd be better if all open source authors include the licence text with the source code or link to it in the README file or something along those lines. In an ideal world. In reality, it is forgotten or we're just lazy and say: `Licence: MIT`. I'm guilty. This repository is. Lots of us are. But it still refers to the MIT licence and it can be argued that - although not strictly compliant with the licence - this should be enough. It's really not that hard to figure out what is meant.

So maybe you could contribute [a year worth of hosting](https://github.com/remy/mit-license#ways-to-contribute) to [this awesome project](https://github.com/remy/mit-license) by Remy Sharp and solve this problem once and for all. :)

**Comment by djmattyg007** [comment]
...I don't understand what is so difficult with just committing the license text. It'll take less than five minutes.

**Comment by hongaar** [comment]
Human psyche, interesting stuff. I don't understand it either :smile:
